rewrite-java/src/main/java/org/openrewrite/java/tree/Javadoc.java

```diff
@@ -298,6 +298,7 @@ public String printComment() {
         UUID id;
 
         String margin;
+        String endOfLine;
```

Comment:
This may be a boolean or enum to limit the values to "\r\n" and "\n", but I wanted to get input first.

The value is separated from the margin to simplify whitespace formatting.